src/MAUI/Maui.Samples/Samples/Location/LocationWithNMEA/LocationWithNMEA.xaml.cs

```diff
@@ -57,33 +55,29 @@ private async Task Initialize()
             // iOS:
             // When using an NMEA device on iOS, use the `NmeaLocationDataSource.FromAccessory` constructor. https://developers.arcgis.com/net/api-reference/api/ios/Esri.ArcGISRuntime/Esri.ArcGISRuntime.Location.NmeaLocationDataSource.FromAccessory.html
 
-            // Set the location data source to use the stream from the simulator.
-            Stream messageStream = _simulatedNMEADataSource.MessageStream;
-            _nmeaSource.NmeaDataStream = messageStream;
-
             // Create an event handler to update the UI when the location changes.
             _nmeaSource.SatellitesChanged += SatellitesChanged;
             _nmeaSource.LocationChanged += LocationChanged;
+            _nmeaSource.SentenceReceived += UpdateNmeaMessageLabel;
 
-            // Start the location data source.
+            // Load the map and assign the location datasource.
```

Comment:
data source should be two words when used in natural language sentences / comments